diff --git a/arch/arm64/boot/dts/amlogic/meson-gxl-s905x-hg680-p.dtsi b/arch/arm64/boot/dts/amlogic/meson-gxl-s905x-hg680-p.dtsi
index b4e1412bb..479fbe85b 100644
--- a/arch/arm64/boot/dts/amlogic/meson-gxl-s905x-hg680-p.dtsi
+++ b/arch/arm64/boot/dts/amlogic/meson-gxl-s905x-hg680-p.dtsi
@@ -190,7 +190,7 @@ &uart_AO {
 
 &usb {
 	status = "okay";
-	dr_mode = "host";
+	dr_mode = "peripheral";
 };
 
 &usb2_phy0 {
